### PR TITLE
Ap add schism simulator

### PR DESCRIPTION
**Pull request by AugustoPeres**
Adds schism simulator to the front end.

### CODE REVIEW

File: inductiva/simulators/schism.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        super().__init__()
+        self.api_method_name = "schism.schism.run_simulation"
+
+    def run(self,
+            input_dir: types.Path,
+            num_scribes: int = 1,
+            on: Optional[types.ComputationalResources] = None,
+            storage_dir: Optional[types.Path] = "",
+            extra_metadata: Optional[dict] = None,
+            n_vcpus: int = None) -> tasks.Task:
```
Comment (Pbarbosa92) [inline inductiva/simulators/schism.py:21]: No hwthreads?

**Review by IvanPombo — CHANGES_REQUESTED** [review]
Add the `hwthreads` and verify it works in the backend. Moreover, since we will perform a deployment soon, this should have a minimal README.md file in the docs.

**Review by sergiomsantos — APPROVED** [review]

File: inductiva/simulators/schism.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+class SCHISM(simulators.Simulator):
+    """Class to invoke a generic SCHISM simulation on the API."""
+
+    def __init__(self):
+        super().__init__()
+        self.api_method_name = "schism.schism.run_simulation"
+
+    def run(self,
+            input_dir: types.Path,
+            num_scribes: int = 1,
```
Comment (IvanPombo) [inline inductiva/simulators/schism.py:17]: Does this work if n_vcpus is 1?

File: inductiva/simulators/schism.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            on: Optional[types.ComputationalResources] = None,
+            storage_dir: Optional[types.Path] = "",
+            use_hwthread: bool = True,
+            extra_metadata: Optional[dict] = None,
+            n_vcpus: int = None) -> tasks.Task:
+        """Run the simulation.
+        Args:
+            input_dir: Path to the directory of the simulation input files.
+            num_scribes: The num_scribes as per the simulator documentation.
```
Comment (IvanPombo) [inline inductiva/simulators/schism.py:26]: Maybe add here a link for that.

**Review by IvanPombo — APPROVED** [review]
Minor comments. LGTM!

File: inductiva/simulators/schism.py (hunk 2 above)
Comment (AugustoPeres) [reply inductiva/simulators/schism.py:17]: I think it will depend on the simulation

**Pull request merged**